bin/settings/settings.env

```diff
@@ -70,7 +66,6 @@ UAADB_PASSWORD=uaaadmin_password
 UAA_ADMIN_CLIENT_SECRET=admin_secret
 UAA_CC_CLIENT_SECRET=cc_client_secret
 UAA_CLIENTS_CC_ROUTING_SECRET=cc_routing_secret
-UAA_CLIENTS_CC_SERVICE_SECRET=cc_service_secret
```

Comment:
This parameter is still in the role manifest
```
 - name: UAA_CLIENTS_CC_SERVICE_SECRET
    secret: true
    immutable: true
    generator:
      type: Password
    description: Used for generating SSO clients for service brokers.
    required: true
```

No templates uses it tough.
Has to be removed from the manifest as well.

Reply:
You realize that with `fissile show image` failing and showing a validation error.

Reply:
Fixed with commit a64048cb686744dd863810ff78483defc923a11a